dbt/models/proximity/proximity.dist_pin_to_stadium.sql

```diff
@@ -0,0 +1,72 @@
+{{ config(
+    materialized='table',
+    partitioned_by=['year'],
+    bucketed_by=['pin10'],
+    bucket_count=1
+) }}
+
+WITH stadiums AS (
+    SELECT
+        CASE
+            WHEN pin10 = '1420227002' THEN 'Wrigley'
+            WHEN pin10 = '1718201035' THEN 'United Center'
+            WHEN pin10 = '1722110002' THEN 'Soldier Field'
+            WHEN pin10 = '1733400049' THEN 'Guaranteed Rate Field'
+            WHEN pin10 = '1722320018' THEN 'Wintrust Arena'
+            WHEN pin10 = '1717239022' THEN 'UIC Pavilion'
+        END AS stadium_name,
+        pin10 AS pin10_stadium,
+        x_3435 AS x_3435_stadium,
+        y_3435 AS y_3435_stadium,
+        year
+    FROM
+        {{ source('spatial', 'parcel') }}
+    WHERE
+        pin10 IN (
+            '1420227002',  -- Wrigley
+            '1718201035',  -- United Center
+            '1722110002',  -- Soldier Field
+            '1733400049',  -- Guaranteed Rate Field
+            '1722320018',  -- Wintrust Arena
+            '1717239022'   -- UIC Pavilion
+        )
+        AND year = '2023'
```

Comment:
Single year in case stadiums change. Would also make sense to do >=. Wintrust Arena is probably the newest at 2017.